include/libdnf/rpm/nevra.hpp

```diff
@@ -118,7 +118,7 @@ template <typename T>
 inline std::string to_full_nevra_string(const T & obj) {
     auto epoch = obj.get_epoch();
     if (epoch.empty()) {
-        epoch = "0";
+        epoch = std::string("0");
```

Comment:
I'm not entirely sure about the objects that can be passed to the function, and I don't understand the weird error message, but assuming `epoch` is of type `std::string`, assigning the char[] literal should be a perfectly fine way of setting its value, so something smells fishy here.

Reply:
I ended up filing a gcc bug: https://bugzilla.redhat.com/show_bug.cgi?id=2057597
I'll remove this commit from the PR and leave only the cmake changes.

Reply:
Thanks! I wonder if we should use your workaround (and add a TODO) to fix the builds for now maybe? Hard to judge how quickly this will be fixed.

Reply:
I've posted https://github.com/rpm-software-management/libdnf/pull/1454 to fix this, using pragmas to locally disable the warning.